inf-clojure.el

```diff
@@ -809,15 +829,16 @@ The prefix argument SWITCH-TO-REPL controls whether to switch to
 REPL after the file is loaded or not.  If the argument FILE-NAME
 is present it will be used instead of the current file."
   (interactive "P")
-  (let ((proc (inf-clojure-proc))
-        (file-name (or file-name
-                       (car (comint-get-source "Load Clojure file: " inf-clojure-prev-l/c-dir/file
-                                               ;; nil because doesn't need an exact name
-                                               inf-clojure-source-modes nil)))))
+  (let* ((proc (inf-clojure-proc))
+         (file-name (or file-name
+                        (car (comint-get-source "Load Clojure file: " inf-clojure-prev-l/c-dir/file
```

Comment:
I would also drop Clojure here, it is kind of clear from the package context ...one day we should actually customize it by repl type (like joker actually loads `.joker` files)